lib/beacon/router_server.ex

```diff
@@ -23,7 +23,7 @@ defmodule Beacon.RouterServer do
     # We store routes by order and length so the most visited pages will likely be in the first rows
     :ets.new(table_name(config.site), [:ordered_set, :named_table, :public, read_concurrency: true])
 
-    if config.mode == :live do
+    if config.mode in [:live, :manual] do
```

Comment:
this was for local testing, the dev script wasn't working without `async_init` being run